### PR TITLE
[ZEPPELIN-5180] - Improve Kerberos debug messages

### PR DESCRIPTION
**Pull request by VipinRathor**
### What is this PR for?
While troubleshooting Kerberos ticket issue, lack of Kerberos log messages makes it difficult to understand what is really happening. This PR aims to address that. It adds debug level log4j messages around ticket renewal thread (in runKerberosLogin()) and in createSecureConfiguration().

### What type of PR is it?
[Improvement]

### Todos
* none

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-5180

### How should this be tested?
* Enable debug level in log4j for Interpreter to see the debug messages

### Screenshots (if appropriate)
* None

### Questions:
* Does the licenses files need update?
No
* Is there breaking changes for older versions?
No
* Does this needs documentation?
No

### CODE REVIEW

**Comment by VipinRathor** [comment]
@prabhjyotsingh @Leemoonsoo @zjffdu please help review. Thanks.

**Comment by VipinRathor** [comment]
Re-based after RAT check hotfix.

**Review by prabhjyotsingh — APPROVED** [review]
LGTM

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java [hunk 1]
```diff
@@ -136,6 +136,8 @@ public Object call() throws Exception {
           logger.info("Ran runKerberosLogin command successfully.");
           kinitFailCount = 0;
           // schedule another kinit run with a fixed delay.
+          logger.info("Scheduling Kerberos ticket refresh thread with interval " +
```
Comment (zjffdu) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java:139]: Could you help to rename `logger` to `LOGGER` ? It is our convention for naming static variables.

Comment (Reamer) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java:139]: The keyword "final" is also desired.

Comment (VipinRathor) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java]: Done.
1. logger is replaced with LOGGER
2. LOGGER is now declared final

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java [hunk 2]
```diff
@@ -106,7 +106,7 @@ private Integer kinitFailThreshold() {
 
   private Long getTimeAsMs(String time) {
     if (time == null) {
-      logger.error("Cannot convert to time value.", time);
+      LOGGER.error("Cannot convert to time value.", time);
```
Comment (Reamer) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java:109]: 'time' makes no sense with this output.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java [hunk 3]
```diff
@@ -133,17 +133,19 @@ private ScheduledExecutorService startKerberosLoginThread() {
       public Object call() throws Exception {
 
         if (runKerberosLogin()) {
-          logger.info("Ran runKerberosLogin command successfully.");
+          LOGGER.info("Ran runKerberosLogin command successfully.");
           kinitFailCount = 0;
           // schedule another kinit run with a fixed delay.
+          LOGGER.info("Scheduling Kerberos ticket refresh thread with interval " +
```
Comment (Reamer) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java:139]: Please use the parameterized method.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java [hunk 4]
```diff
@@ -133,17 +133,19 @@ private ScheduledExecutorService startKerberosLoginThread() {
...
+            getKerberosRefreshInterval() + " ms");
           scheduledExecutorService
               .schedule(this, getKerberosRefreshInterval(), TimeUnit.MILLISECONDS);
         } else {
           kinitFailCount++;
-          logger.info("runKerberosLogin failed for " + kinitFailCount + " time(s).");
+          LOGGER.info("runKerberosLogin failed for " + kinitFailCount + " time(s).");
```
Comment (Reamer) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java:145]: Please use the parameterized message.

**Review by Reamer — CHANGES_REQUESTED** [review]
Thank you for improving the code. Just two small comments.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java (hunk 2 above)
Comment (VipinRathor) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java]: Fixed the error message.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java (hunk 3 above)
Comment (VipinRathor) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java]: Done

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java (hunk 4 above)
Comment (VipinRathor) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/KerberosInterpreter.java]: Done

**Comment by VipinRathor** [comment]
@Reamer Thanks for your review comments. I just pushed the changes. Please help review.

**Review by Reamer — APPROVED** [review]

**Pull request closed without merging**